topology.go

```diff
@@ -138,7 +142,7 @@ func (gf *generationFetcher) tryFetchGenerations() {
 	}
```

Comment:
BTW. you should not be fetching generations with CL=ALL because it doesn't tolerate any failures (one of the replicas is down and you can't fetch generations anymore). Switching to CL=QUORUM improves availability without losing consistency (the table is written to using QUORUM as well).

Reply:
OK, as a bonus I changed the consistency choosing algorithm. Now, if the cluster has only one node, it uses ONE, otherwise it uses QUORUM.